web/ui/mantine-ui/src/pages/AlertsPage.tsx

```diff
@@ -162,10 +162,10 @@ export default function AlertsPage() {
     withDefault(StringParam, "")
   );
   const [debouncedSearch] = useDebouncedValue<string>(searchFilter.trim(), 250);
-  const [showEmptyGroups, setShowEmptyGroups] = useQueryParam(
-    "showEmptyGroups",
-    withDefault(BooleanParam, true)
-  );
+  const [showEmptyGroups, setShowEmptyGroups] = useLocalStorage<boolean>({
+    key: "alerts-page-show-empty-groups",
```

Comment:
Nice. Could you follow the same naming scheme as for the other localStorage keys though? Like `alertsPage.showEmptyGroups` here for example. Similar for the targets page below.

Reply:
done, I think I saw some keys from the /react-app that were like `targets-page..`. Just realized that this is another front end.

Reply:
hey, did you have the chance to take another look?